tests/unit_tests/experiment_plans/test_flyscan_xray_centre_plan.py

```diff
@@ -14,6 +14,7 @@
 from dodal.devices.fast_grid_scan import FastGridScan
 from ophyd.sim import make_fake_device
 from ophyd.status import Status
+from ophyd_async.core import set_sim_value
 
 from hyperion.device_setup_plans.read_hardware_for_setup import (
     read_hardware_for_ispyb_during_collection,
```

Comment:
double import of `read_hardware_for_ispyb_pre_collection` below

Reply:
Spent a good hour trying to work out why this wasn't picked up by the static analysis. It looks like it's generally a thing that our tools are happy with :/